### ISSUE TITLE
A11y_code.visualstudio.com_Docs Azure_Try Azure App Service_ScreenReader: Images present in the screen does not contain alternative text.

### ISSUE THREAD

**Issue by Charanrajkb** [microsoft/vscode-docs#2801]
“[Check out Accessibility Insights!](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
URL: [https://code.visualstudio.com](url)
Windows Version: Windows10
Browser Version:
Microsoft Edge 44.18362.1.0
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Screen Reader Versions : NVDA2019.1.1
MAS Violated: MAS 1.1.1

**Repro Steps:**
Navigate to the URL: [https://code.visualstudio.com/tryappservice/](url)
or
Follow below steps:
1)Launch the application Url : [https://code.visualstudio.com](url)
2)Navigate to "Docs" link.
3)From left navigation, select "Azure'.
4)Navigate to "Try Azure App Service" link.
5)Start Narrator.
6)Navigate using the Caps + arrow Key till the image.
7)Observe the narrator reading.

**Actual:**
Images present in the screen does not contain alternative text.
Screen reader just reads as images which is not descriptive.

**Issue is observed in Edge + Narrator and Chrome + NVDA.**

**Expected:**
Images must contain the alternative text for the images present in the screen.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](url)

**User Impact**:
Screen reader users will find very difficult in understand the images if the alternative text is not given.

**MAS Reference:**
MAS 1.1.1 -
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2d2051f-bdc8-4af7-8e18-38aae867e216}](url)

**Attachment for Reference:**
[A11y_code.visualstudio.com_Docs Azure_Try Azure App Service_ScreenReader1.pptx](https://github.com/microsoft/vscode-docs/files/3320681/A11y_code.visualstudio.com_Docs.Azure_Try.Azure.App.Service_ScreenReader1.pptx)

**Comment by bowdenk7** [comment]
fixed by https://github.com/microsoft/TryAppServiceClient/commit/efe8079ef62fe271c6deeba0187fefede643c4f7

**Comment by SwatiTiwari-TCS** [comment]
"The issue is still reproducing can you please open the issue".

**Comment by gregvanl** [comment]
This is a low page view area and requires changes to an external repo. We asked that testing not be done on this page.
